src/query/ts/m3db/consolidators/step_consolidator.go

```diff
@@ -108,6 +106,7 @@ func (c *StepLookbackConsolidator) ConsolidateAndMoveToNext() float64 {
 	}
 
 	val := c.unconsumed[0]
-	c.unconsumed = c.unconsumed[1:]
+	copy(c.unconsumed, c.unconsumed[1:])
+	c.unconsumed = c.unconsumed[:len(c.unconsumed)-1]
```

Comment:
Similarly can you remove changes to this file? Coming here:
https://github.com/m3db/m3/pull/2185